web/impact/impact/v1/helpers/organization_helper.py

```diff
@@ -0,0 +1,67 @@
+from impact.models import Organization
+from impact.v1.helpers.model_helper import ModelHelper
+
+
+class OrganizationHelper(ModelHelper):
+    MODEL = Organization
+
+    REQUIRED_KEYS = [
+        "name",
+        "url_slug",
+        ]
+    OPTIONAL_KEYS = [
+        "additional_industry_ids",
+        "date_founded",
+        "facebook_url",
+        "full_elevator_pitch",
+        "primary_industry_id",
+        "public_inquiry_email",
+        "linked_in_url",
+        "location_city",
+        "location_national",
+        "location_postcode",
+        "location_regional",
+        "short_pitch",
+        "twitter_handle",
+        "video_elevator_pitch_url",
+        "website_url",
+        ]
+    INPUT_KEYS = REQUIRED_KEYS + OPTIONAL_KEYS
+    READ_ONLY_KEYS = [
+        "id",
+        "is_partner",
+        "is_startup",
```

Comment:
again - we should probably be inferencing as many readonly fields as we can directly from the model rather than hardcoding lists of keys

Reply:
See comment at the end.